3.0-RELEASE-NOTES.md

```diff
@@ -127,3 +127,16 @@ removed properties together with the middleware module name to use instead:
 We have also removed `loopback.mime`, which was always set to `undefined`.
 
 See [loopback#2349](https://github.com/strongloop/loopback/pull/2394).
+
+## remove `loopback.errorhandler`
```

Comment:
## Remove `loopback#errorhandler`

We have removed `loopback#errorhandler` middleware, users should use `strong-error-handler` directly.

```js
// server/middleware.json
{
  // ...
  "final:after": {
    "strong-error-handler": {}
  }
}

// server/middleware.development.json
{
  "final:after": {
    "strong-error-handler": {
      "params": {
        "debug": true
      }
  }
}
```

See also strong-error-handler's [options](https://github.com/strongloop/strong-error-handler#options) and the [related code change](https://github.com/strongloop/loopback/pull/2411)